### PR TITLE
unification audit

### PR DESCRIPTION
**Pull request by cowboy**
It will be easier for @emilyrohrbough to make suggestions here in a separate PR than in https://github.com/cypress-io/cypress-documentation/pull/4186

Process:
- @emilyrohrbough will make suggestions in comments on this draft PR
- When she's done, we will batch them all into a commit
- We'll follow that up with a `yarn fix:markdown` commit
- We'll update this PR's base from `master` to `unification`
- We'll actually review the changes

### CODE REVIEW

File: content/guides/core-concepts/writing-and-organizing-tests.md [hunk 1]
```diff
@@ -88,16 +106,16 @@ Cypress will create a
...
 these should also be ignored when you check into source control.
 
 </Alert>
 
 ### Test files
 
-Test files are located in `cypress/integration` by default, but can be
+Test files are located in `cypress/e2e` by default, but can be
 [configured](/guides/references/configuration#Folders-Files) to another
 directory. Test files may be written as:
```
Comment (emilyrohrbough) [inline content/guides/core-concepts/writing-and-organizing-tests.md:120]: This needs to be updated to be to:
```suggestion
directory. Test files may be written as:

- `.js`
- `.jsx`
- `.ts`
- `.tsx`
```

**Comment by emilyrohrbough** [comment]
FYI: there are multiple places in the docs like [here](https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/guides/core-concepts/writing-and-organizing-tests#Support-file), that link to the supportFile configuration key, however, it directs to the [Folder-Files configuration section](https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/guides/references/configuration#Folders-Files) which doesn't make sense anymore since this is not longer listed under this section.

**Comment by emilyrohrbough** [comment]
Seems like this section might needs updated to reflect setupNodeEvents? This is another weird on where it could be plugins but isn't.
https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/guides/guides/environment-variables#Option-5-Plugins

**Comment by emilyrohrbough** [comment]
https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/guides/core-concepts/writing-and-organizing-tests#Test-files Need to verify that `.csj` 

[Hover API documentation](https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/api/commands/hover#Add-as-a-custom-command) and [Hover API docs](https://cypress-documentation-h2l08ulip-cypress-io.vercel.app/api/commands/mount#React-Router) needs updated to reflect a custom hover/mount commands can now be added with `Cypress.Commands.add` instead of `Cypress.Commands.override`. See https://github.com/cypress-io/cypress/pull/19633

**Comment by emilyrohrbough** [comment]
FYI, we now support additional cypress configuration file extensions by default: https://github.com/cypress-io/cypress/pull/20554

File: content/guides/core-concepts/writing-and-organizing-tests.md (hunk 1 above)
Comment (elylucas) [reply content/guides/core-concepts/writing-and-organizing-tests.md:120]: [17bd0f2](https://github.com/cypress-io/cypress-documentation/commit/17bd0f21aa3c3eb20cc204fc3d0fa27c950c06a5)

File: content/guides/core-concepts/writing-and-organizing-tests.md [hunk 2]
```diff
@@ -88,16 +106,16 @@ Cypress will create a
...
-are storing sensitive environment variables in your configuration file
-(`cypress.json` by default) or
+are storing sensitive environment variables in your
+[Cypress configuration](/guides/references/configuration) or
 [`cypress.env.json`](/guides/guides/environment-variables#Option-2-cypress-env-json),
 these should also be ignored when you check into source control.
 
 </Alert>
 
 ### Test files
```
[the remark this answers is not in the document]
Comment (elylucas) [reply content/guides/core-concepts/writing-and-organizing-tests.md:116]: [f7aefb9](https://github.com/cypress-io/cypress-documentation/commit/f7aefb9895c328e912504ac842c35ce0fab1eb9a)

File: content/guides/core-concepts/writing-and-organizing-tests.md [hunk 3]
```diff
@@ -210,19 +263,17 @@ The initial imported plugins file can be
 
 ### Support file
 
-By default Cypress will automatically include the support file
-`cypress/support/index.js`. This file runs **before** every single spec file. We
-do this purely as a convenience mechanism so you don't have to import this file
-in every single one of your spec files.
+By default Cypress will automatically include type-specific support files. For
+E2E, the default is `cypress/support/e2e.{js,jsx,ts,tsx}`, and for Component
+Testing `cypress/support/e2e.{js,jsx,ts,tsx}`.
```
[the remark this answers is not in the document]
Comment (elylucas) [reply content/guides/core-concepts/writing-and-organizing-tests.md:268]: [218633d](https://github.com/cypress-io/cypress-documentation/commit/218633d7f7ae819351d36424685301a893890d9d)

**Comment by elylucas** [comment]
> 20 days ago

[5612d15](https://github.com/cypress-io/cypress-documentation/commit/5612d15d237b02ebcf851b1ca4c5d7fb1dbee57a)

**Comment by cowboy** [comment]
Closing in favor of #4412

**Pull request closed without merging**